[FEAT] 프로젝트 초기 설정

dpdms529 opened the pull request:
# 📄 Work Description
- 프로젝트 초기 설정

# ⚙️ ISSUE
- closed #1 


# 📷 Screenshot
## 패키지 구조
![image](https://github.com/user-attachments/assets/f7e53897-30d7-40d2-9872-699cd8c4f32a)
![image](https://github.com/user-attachments/assets/f7579015-c15c-4866-a401-80ad55b77046)

## 테이블
![image](https://github.com/user-attachments/assets/a50eda6a-e8bd-41b4-8813-2cbf98d78aa7)
![image](https://github.com/user-attachments/assets/c3aec2d0-620f-4a95-aeaa-713e11a6a6ed)
![image](https://github.com/user-attachments/assets/f6176b57-8b89-4295-8461-2e66c7289999)
![image](https://github.com/user-attachments/assets/67cddb4a-2a63-4d75-ab46-4b7fb42b12bc)
![image](https://github.com/user-attachments/assets/50e06629-00a0-4fcf-94e9-d19284ad4f0a)
![image](https://github.com/user-attachments/assets/4a5698ce-1046-4ecd-b687-49333fc25972)
![image](https://github.com/user-attachments/assets/e12faca6-fd93-48c8-b317-9f0b97922d44)

## 예외처리
![image](https://github.com/user-attachments/assets/3848ec19-9ae6-4f1c-affc-94ce129ce545)


# 💬 To Reviewers
너무 힘들어요...

dpdms529 committed:
[feat/#1]: gitignore 설정

application.yml 모두 무시하도록 설정
application-dev.yml,application-prod.yml 만 올라가도록 설정

dpdms529 committed:
Merge branch 'main' into feat/#1-init

dpdms529 committed:
[feat/#1]: 컨트롤러 생성

dpdms529 committed:
[feat/#1]: 서비스 생성

dpdms529 committed:
[feat/#1]: 엔티티 생성

아이디만 가지고 있는 초기 상태

dpdms529 committed:
[feat/#1]: Account 엔티티 및 레포지토리 생성

dpdms529 committed:
[feat/#1]: AccountTransaction 엔티티 및 레포지토리 생성

dpdms529 committed:
[feat/#1]: BaasMember 엔티티 및 레포지토리 생성

dpdms529 committed:
[feat/#1]: BankMember 엔티티 및 레포지토리 생성

dpdms529 committed:
[feat/#1] .gitignore 수정

dpdms529 committed:
[feat/#1]: Product 엔티티 및 레포지토리 생성

dpdms529 committed:
[feat/#1]: Bank 삭제

dpdms529 committed:
[feat/#1]: Card 엔티티 및 레포지토리 생성

dpdms529 committed:
[feat/#1]: dto 임시 파일 생성

khwoowoo commented on src/main/java/com/wekids/baas/accountTransaction/domain/AccountTransaction.java:
```diff
@@ -0,0 +1,49 @@
+    @Column(nullable = false)
+    private LocalDateTime transactionDate;
+
+    @Column(nullable = false)
+    private CurrencyCode currencyCode;
```
@Enumerated(EnumType.STRING) 이거 추가해야 할 것 같습니다!

khwoowoo commented on src/main/java/com/wekids/baas/accountTransaction/domain/AccountTransaction.java:
```diff
@@ -0,0 +1,49 @@
+    private String title;
+
+    @Column(nullable = false)
+    private AccountTransactionType type;
```
이것도요!

khwoowoo commented on src/main/java/com/wekids/baas/accountTransaction/domain/enums/AccountTransactionType.java:
```diff
@@ -0,0 +1,4 @@
+package com.wekids.baas.accountTransaction.domain.enums;
+
+public enum AccountTransactionType {
+}
```
비워두신 이유가 있을까요>??

dpdms529 committed:
[feat/#1]: 예외 처리

khwoowoo reviewed (commented):
@EnableJpaAuditing
이거 Applicaation 파일에 추가하셔야합니다!
그래야지 BaseTime이 동작합니다

그리고 Bank 엔티티는 없는건가요??

dpdms529 committed:
[feat/#1]: 코드 리뷰 해결

- WeKidsBaaSApplication에 @EnableJpaAuditing 추가
- Enum 타입 컬럼에 @Enumerated(EnumType.STRING) 추가
- AccountTransactionType 내용 작성

cbj0010 reviewed (approved):
I will trust you

khwoowoo reviewed (approved):
수고하셨습니다 👍
